### PR TITLE
test: add unit tests for create new password form validation and rese…

### PR DESCRIPTION
**Pull request by umbereye27**
This PR introduces unit tests to ensure correctness of the Create New Password form validation and the Reset Password flow.

- Added validation tests for required fields, password length, and confirmation match

- Covered edge cases for invalid and valid inputs

- Ensured reset password flow behaves as expecte

### CODE REVIEW

File: __tests__/auth/create-password-form.test.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        screen.getByText("Must contain uppercase letter")
+      ).toBeInTheDocument();
+      expect(
+        screen.getByText("Must contain lowercase letter")
+      ).toBeInTheDocument();
+      expect(screen.getByText("Must contain number")).toBeInTheDocument();
+    });
+  });
+
+  //   it("shows error on API failure and does not show success message", async () => {
```
Comment (Bob-Karemera-Shema) [inline __tests__/auth/create-password-form.test.tsx]: Consider removing commented tests

File: app/(auth)/reset-password/page.tsx [hunk 2]
```diff
@@ -3,6 +3,19 @@ import { ResetPasswordForm } from "./component/reset-password-form"
 export default function ResetPasswordPage() {
   return (
     <>
+        {/* <div
```
Comment (Bob-Karemera-Shema) [inline app/(auth)/reset-password/page.tsx]: Consider removing commented code here as well

**Review by Bob-Karemera-Shema — CHANGES_REQUESTED** [review]
Overall, the code looks good, just minor issues left to make the code cleaner.

**Review by Bob-Karemera-Shema — DISMISSED** [review]

**Review by Favor-star — DISMISSED** [review]

**Review by Bob-Karemera-Shema — APPROVED** [review]
